sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ### Features Added
 
+- Added `AnalyzeActionsResult.AnalyzeHealthcareEntitiesResults` to get healthcare entities from a batch of actions.
```

Comment:
[super nit] Is it just me who found it odd to read Healthcare vs HealthCare

Reply:
This came up in EventGrid. I believe Healthcare is the correct casing based on how we have branded it - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs#L123

Reply:
"healthcare" is one word and used as such by the service throughout.